nifi-nar-bundles/nifi-framework-bundle/nifi-framework/nifi-web/nifi-web-ui/src/main/java/Messages_cn.properties

```diff
@@ -0,0 +1,785 @@
+#-------------------------------------------------
```

Comment:
@wangrenlei Travis CI reported [RAT check error](https://travis-ci.org/apache/nifi/jobs/212601713) as below:

```
[ERROR] Failed to execute goal org.apache.rat:apache-rat-plugin:0.11:check (default-cli) on project nifi-web-ui: Too many files with unapproved license: 5 See RAT report in: /home/travis/build/apache/nifi/nifi-nar-bundles/nifi-framework-bundle/nifi-framework/nifi-web/nifi-web-ui/target/rat.txt -> [Help 1]
```

I haven't checked out this PR yet, but suspect these RAT errors came from newly added properties files (or other files). Because those don't have appropriate license note in it. Please add Apache 2 License note at the head of each files. [nifi.properties](https://github.com/apache/nifi/blob/master/nifi-nar-bundles/nifi-framework-bundle/nifi-framework/nifi-resources/src/main/resources/conf/nifi.properties) can be a reference for doing that.

Then please make sure mvn build passes with `contrib-check` profile as written in the [Contributor Guide](https://cwiki.apache.org/confluence/display/NIFI/Contributor+Guide#ContributorGuide-Providingcodeordocumentationcontributions):

```
# Test your changes
For code changes, ensure that the full suite of tests is executed via mvn -Pcontrib-check clean install at the root nifi folder.  Please write or update unit tests to verify your changes.
Did your change introduce new dependencies?  Are these dependencies licensed in a way that is compatible for inclusion under ASF 2.0?  Did you update the LICENSE file?
For documentation related changes, ensure that format looks appropriate for the output in which it is rendered.
```